src/components/D3Rings.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div />
+</template>
+<script>
+import mapboxgl from "mapbox-gl";
+import * as d3 from "d3";
+import tour from "../assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted";
+export default {
+  name: "D3Rings",
+  mounted() {
+    this.$root.$on("CreateRings", () => {
+      this.CreateRings();
+    });
+    this.$root.$on("RemoveRings", () => {
+      this.RemoveRings();
+    });
+  },
+  methods: {
+    CreateRings() {
+      let map = this.$store.map;
+      let $this = this;
+      let radius = 15;
+      //Gage array entries match up to the index of the color array
+      let gageArray = ["enhanced_gage", "new_gage", "specific_conductance", "temperature", "camera"];
```

Comment:
To start. SO COOL, love this. Much less code than I imagined. Second, just a few minor things. Can we use the term 'monitoringLocation' in the place of 'gage'? That will be more consistent with the rest of the code. And 'gage' is just a dumb spelling anyway :). Third, I think we can grab the array using object.keys ( this would probably be best if we modified the JSON, so that the location features are not in with all the other 'properties' but grouped in their own array or object. Changing that will break some things so maybe I should just make a separate ticket for that). But then if we change the JSON to add more features it won't break the code. Yeah, so thinking about that, let's just make an new issue for that.

Reply:
Thats why I was asking about the JSON yesterday, was hard to do things with all those other objects just sitting there being useless for my purposes.  Good thinking.